script/release.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+WORKDIR=$(pwd)
+SVU_BIN="${WORKDIR}/svu"
+
+echo "+++ :construction:  Installing 'svu' tool"
+curl -sfL https://install.goreleaser.com/github.com/caarlos0/svu.sh | bash -s -- -b $WORKDIR
+
+# TODO: change patch to minor
+RELEASE_VERSION=$($SVU_BIN patch)
```

Comment:
I'm planning a follow-up PR to change it after testing the release pipeline on the main branch.